src/main/webapp/i18n/de/exam.json

```diff
@@ -93,7 +93,7 @@
             "secondCorrectionColumn": "Zweitkorrektur",
             "titleExerciseGroupColumn": "Aufgabengruppe",
             "titleExerciseColumn": "Aufgabe",
-            "averagePointsColumn": "Durchschnitt",
+            "averagePointsColumn": "Durchschnittspunktzahl",
```

Comment:
Maybe "Durchschnittliche Punktzahl" (it's also Durchschnittliche Note / Bonuspunkte)

Reply:
Yes, sounds more consistent 👍